sqlite-android/src/main/java/io/requery/android/database/CursorWindow.java

```diff
@@ -76,14 +85,29 @@
      * </p>
      *
      * @param name The name of the cursor window, or null if none.
+     * @param windowSizeBytes Size of cursor window in bytes.
+     *
+     * Note: Memory is dynamically allocated as data rows are added to
+     * the window. Depending on the amount of data stored, the actual
+     * amount of memory allocated can be lower than specified size,
+     * but cannot exceed it. Value is a non-negative number of bytes.
      */
-    public CursorWindow(String name) {
+    public CursorWindow(String name, int windowSizeBytes) {
```

Comment:
Might be worth noting also the upstream API is API >= 28 only, so if people start using this API they'll have to use requery or move their minSdkVersion to 28